sdk/netapp/Azure.ResourceManager.NetApp/api/Azure.ResourceManager.NetApp.netstandard2.0.cs

```diff
@@ -601,6 +617,19 @@ public partial class AvailabilityZoneMapping
         public string AvailabilityZone { get { throw null; } }
         public bool? IsAvailable { get { throw null; } }
     }
+    public partial class BackupRestoreFiles
+    {
+        public BackupRestoreFiles(System.Collections.Generic.IEnumerable<string> fileList, string destinationVolumeId) { }
+        public string DestinationVolumeId { get { throw null; } }
```

Comment:
Is the value of this property an arm resource identifier?
If so, please change the format of this property to `arm-id`